pkg/graveler/graveler.proto

```diff
@@ -50,8 +50,9 @@ message GarbageCollectionRules {
 
 message GarbageCollectionRunMetadata {
   string run_id = 1;
-  string Commits_csv_location = 2;
-  string Address_location = 3;
+  string commits_csv_location = 2;
+  string address_location = 3;
+  string uncommitted_location = 4;
 }
```

Comment:
It's hard to understand these fields without documentation. `address_location`? I know you just added 1 field but I think we need to add some info here.